src/lib/plugin_apis/swap.api

```diff
@@ -99,3 +100,15 @@ gboolean bd_swap_swapstatus (const gchar *device, GError **error);
  * Tech category: %BD_SWAP_TECH_SWAP-%BD_SWAP_TECH_MODE_SET_LABEL
  */
 gboolean bd_swap_set_label (const gchar *device, const gchar *label, GError **error);
+
+/**
+ * bd_swap_set_uuid:
```

Comment:
Needs to be added to `docs/libblockdev-sections.txt`?

Reply:
Fixed.